osbs/cli/main.py

```diff
@@ -313,11 +315,20 @@ def _print_build_logs(args, osbs, build):
         print("Build submitted (%s), watching logs (feel free to interrupt)" % build_id)
         try:
             for line in build_logs:
+                if "- USER_WARNING -" in line:
```

Comment:
:eyes:  Why a hardcoded magic value? Don't we have defined constant for that somewhere?